xchange-idex/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
+        <version>4.3.6-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>xchange-idex</artifactId>
+
+    <name>XChange Idex</name>
+    <description>XChange implementation for the Idex Exchange</description>
+
+    <url>http://knowm.org/open-source/xchange/</url>
+    <inceptionYear>2018</inceptionYear>
+
+    <organization>
+        <name>Knowm Inc.</name>
+        <url>http://knowm.org/open-source/xchange/</url>
+    </organization>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.knowm.xchange</groupId>
+            <artifactId>xchange-core</artifactId>
+            <version>4.3.6-SNAPSHOT</version>
+        </dependency>
+        <dependency>
```

Comment:
This should go in the root pom.xml under dependencyManagement. Here, just remove the version.

Reply:
this was from huobi, the one before in the order, for consistency

Reply:
I meant the org.web3j one :)